src/apis/exh/content/draft/index.ts

```diff
@@ -0,0 +1,39 @@
+import { AuthToken } from "apis/@types";
+import { Draft, DraftParameter, NewDraft } from "./@types";
+
+export interface Methods {
+  /**
+   * draft のリストを取得する
+   *
+   * @remarks
+   * 必要な権限:
+   * - exh : 自身が作成した draft のみ
+   * - blogAdmin : 全ての draft
+   *
+   * @returns 条件を満たす draft の配列
+   */
+  get: {
+    reqHeaders: AuthToken;
+    resBody: Draft[];
+    query?: DraftParameter;
+  };
+
+  /**
+   * draft を新規作成する
+   * 既に waiting の draft がある場合は先にある方が reject される
+   *
+   * @remarks
+   * 必要な権限:
+   * - exh
+   *
+   * @throws Error
+   * 400: パラメータに過不足がある
```

Comment:
余計なものがあってもエラーにはならない (言葉の綾なんだろうけど、気になる)
```suggestion
   * 400: 不正なパラメータ
```

Reply:
`パラメータに不備がある` で統一されてるっぽい